src/main/webapp/js/const.es6

```diff
@@ -3,6 +3,32 @@
  */
 const Const = {
 
+    Selectors: {
+
+        // Selector for status message div tag (to be used in jQuery)
+        DIV_STATUS_MESSAGE: '#statusMessagesToUser',
```

Comment:
Actually, if this is only used in one class, just move it to `statusMessage.es6`. Otherwise it looks weird that there is a sub-class for `Selectors` with only one content.